src/avt/Pipeline/Data/avtCommonDataFunctions.C

```diff
@@ -1491,6 +1497,9 @@ GetNodalScalarRangeViaCellsTemplate(Array buf, Scalar &min, Scalar &max,
                 if (! visitIsFinite(buf[id]))
                     continue;
 
+            if (std::isnan(buf[id]))
```

Comment:
The logic just above, the test for finiteness, is predicated on 'checkFinite' being true.
Comments for the function indicate it was done this way to speed up the function.
At the end of the function, if the min or max value was determined to be not finite, then the method was called again with the checkFinite flag set to true.
I think the same logic should be applied for the isnan check.

Reply:
I  tried to build on Windows, and get 'ambiguous' errors for std::isnan in the templated functions GetScalarRangeTemplate and GetNodalScalarRangeViaCellsTemplate.  Not sure the best way to fix:  using windows '_isnan' function resolves the error, as does casting the value to double.

Reply:
Sounds good. I'll look into this and make changes next week.

Reply:
Okay, I updated the nan check to follow the finite check logic. I also updated the nan check to be OS sensitive. Can you let me know if this change resolved that issue when you get a chance?

Reply:
Looks good.